src/content/docs/opentelemetry/best-practices/opentelemetry-manage-ingest-volume.mdx

```diff
@@ -0,0 +1,233 @@
+
+See "SDK resource detectors", "Collector resource processor", and "Collector transform processor" in the [tool catalog](#tool-catalog).
+
+## Key concepts: Traces [#concept-traces]
+
+### Sampling [#concept-sampling]
+
+[Sampling](https://opentelemetry.io/docs/concepts/sampling/) is the process of controlling which spans are exported to an observability backend. Trace data can provide highly valuable insights, but if unchecked, can quickly fill up hard drives (and bills!).
+
+By default, OpenTelemetry SDKs use the [`ParentBased(root=AlwaysOn)`](https://opentelemetry.io/docs/specs/otel/trace/sdk/#parentbased) sampler. The `ParentBased` sampler delegates to different configurable samplers based on whether there is a parent span, whether that parent is local to the current process or remote, and whether that parent is sampled. The default `ParentBased(root=AlwaysOn)` sampler will sample a span if either of the following is true:
+
+- There is no parent span (i.e. this span is the root)
+- The parent is sampled, regardless of whether the parent is local or remote
+
+In other words, it will sample the span unless the parent is unsampled.
+
+This is a good default for the OpenTelemetry community since it allows users to install instrumentation and see trace data without first needing to be aware of the sampling concept. However, production deployments of OpenTelemetry should be cautious. Under this policy, **all spans** are sampled unless there is some upstream component or gateway making intelligent sampling decisions for downstream systems to conform to.
+
+See "ParentBased(root=TraceIdRatioBased) sampler", "Collector filter processor", and "Collector tailsampling processor" in the [tool catalog](#tool-catalog) for alternatives.
+
+### Span data [#concept-span-data]
+
+OpenTelemetry spans are composed of a variety of top level fields (name, kind, etc.), attributes, span events, and span links. The amount of data attached to spans directly corresponds to data volume in New Relic.
+
+Instrumentation libraries make decisions about what pieces of information to attach to spans, often following the [semantic conventions](https://opentelemetry.io/docs/specs/semconv/). When exceptions occur, the information is often attached to the span in the form of an [exception span event](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/). This event includes attributes representing the exception message, type, and stacktrace, which depending on the language and application, can consist of thousands of bytes. If exceptions occur frequently, stacktraces may produce high data volume in New Relic.
```

Comment:
I've always seen stacktrace as a single word but wikipedia suggests otherwise! https://en.wikipedia.org/wiki/Stack_trace#:~:text=In%20computing%2C%20a%20stack%20trace,the%20stack%20and%20the%20heap.